pkg/test/verify/cluster.go

```diff
@@ -137,8 +134,8 @@ func AddToolchainClusterFailsBecauseOfEmptySecret(t *testing.T, functionToVerify
 	// given
 	defer gock.Off()
 	status := test.NewClusterStatus(toolchainv1alpha1.ToolchainClusterReady, corev1.ConditionTrue)
-	toolchainCluster, _ := test.NewToolchainCluster("east", "secret", status,
-		Labels("", "", test.NameHost))
+	toolchainCluster, _ := test.NewToolchainCluster("east", test.MemberOperatorNs, "secret", status,
+		Labels(test.MemberOperatorNs, test.NameHost))
```

Comment:
minor - but just for consistency with the secret namespace
```suggestion
	toolchainCluster, _ := test.NewToolchainCluster("east", "test-namespace", "secret", status,
		Labels("test-namespace", test.NameHost))
```

Reply:
no the secret namespace  should be changed to "test.MemberOperatorNs" , i have updated it now